src/environments/.env.ts

```
@@ -2,8 +2,8 @@
 /* tslint:disable */
 export default {
   'mifos_x': {
-    'version': '260108',
-    'hash': 'g5af11fe75'
+    'version': '260311',
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The 'version': '260311', key has incorrect delimiter

(IncorrectDelimiter)